user_flair/esports_flair.scss

```diff
@@ -94,3 +94,51 @@
 .flair-e-mighty::before {
   background-position: 0 -400px;
 }
+
+.flair-e-2arc::before {
+  background-position: 0 -420px;
+}
+
+.flair-e-alternate::before {
+  background-position: 0 -440px;
+}
+
+.flair-e-navi::before {
+  background-position: 0 -460px;
+}
+
+.flair-e-complexity::before {
+  background-position: 0 -480px;
+}
+
+.flair-e-EG::before {
```

Comment:
While we're moving these over, we may as well make them all lowercase, as CSS is case insensitive.